source/Halibut.Tests/Support/BackwardsCompatibility/CompatBinaryStayAlive.cs

```diff
@@ -1,28 +1,30 @@
 using System;
 using System.IO;
+using Serilog;
 
 namespace Halibut.Tests.Support.BackwardsCompatibility
 {
     public class CompatBinaryStayAlive : IDisposable
     {
-
+        readonly ILogger logger;
```

Comment:
Wow fancy

Reply:
Oh sorry I thought this was adding real logging to the external compatibility binary which just does crappy STDOUT logging :p

Reply:
I didn't like the construction of loggers in dispose methods so this is why there is this logging plumbing change